app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
+  def recent_three_posts
+    posts.limit(3).order(created_at: :desc)
+  end
```

Comment:
- [ optional ] Hi you really did great. ill just recommend using the order before the limit method. with your current code, it is going to call for the first 3 elements in the database before ordering it in descending order meanwhile your goal is to call for the last 3 elements in the database
-  i.e `posts.order(created_at: :desc).limit(3)`

Reply:
Hey @m05e5,  Thanks a lot for your brief feedback and review. I will make the optional change